File: SoloLearn Codes/Skee-Ball/SkeeBall.cs
using System;

namespace Sololearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int points = Convert.ToInt32(Console.ReadLine());
            int tickets = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine(points / 12 >= tickets ? "Buy it!" : "Try again");
        }
    }
}

File: SoloLearn Codes/Skee-Ball/SkeeBall.java
import java.util.Scanner;

public class SkeeBall
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int points = sc.nextInt();
        int tickets = sc.nextInt();
        System.out.println(points / 12 >= tickets ? "Buy it!" : "Try again");
    }
}